web/server/fastify/program.ts

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+import { Command, InvalidArgumentError } from "commander";
+const program = new Command();
+import { start } from "./server.js";
+
+function validateNumber(value: string) {
+  const parsedValue = parseInt(value, 10);
+  if (Number.isNaN(value)) {
+    throw new InvalidArgumentError("option must be a number.");
+  }
+  return parsedValue;
+}
+
+program
```

Comment:
The go `serverconnect` and `servergrpc` commands print [server.v1.ServerMetadata](https://github.com/bufbuild/connect-crosstest/blob/0a1902575a342e9cbd62453bb7f832179cff5d11/proto/server/v1/server.proto#L19) at startup. 

Is the output parsed anywhere? :thinking:

Reply:
It doesn't appear to be used anywhere, but I use that same `ServerMetadata` object to print out the information in the server startup for Fastify if you like?

Reply:
Yes, let's stick with the pattern.